crates/re_tuid/Cargo.toml

```diff
@@ -19,8 +19,8 @@ all-features = true
 [features]
 default = []
 
-## Enable converting Tuid to arrow2
-arrow2_convert = ["dep:arrow2", "dep:arrow2_convert"]
+## Enable (de)serialization using Arrow.
+arrow = ["dep:re_types", "dep:arrow2", "dep:thiserror"]
```

Comment:
having `re_tuid` depend on `re_types` feels very backwards to me. `re_tuid` is a very low-level building block. This would be like adding a dependency on `re_types` to the `uuid` crate. I think it makes much more sense to `impl re_types::Loggable for Tuid` in `re_types` instead.

Reply:
Yeah I guess if we start to add these kinds of external integrations into `re_types`, we might as well just split the core traits into their own crate already and avoid the intermediate headache.

I'll do the split in a follow up.